projects/cashmere/src/lib/header/header.component.html

```diff
@@ -3,7 +3,7 @@
         <a *ngIf="homeUriIsRelative()" [routerLink]="homeUri" class="brand">
             <img alt="Brand Icon" src="{{ brandIcon }}"/>
         </a>
-        <a *ngIf="!homeUriIsRelative()" [href]="homeUri" class="brand">
+        <a [href]="homeUri" class="brand">
```

Comment:
This line should still get swapped back to how it was.

Reply:
Good glory where is my head today.  Fixed and recommitted.